### ISSUE TITLE
XS, SM, and MD Button sizes: do we need all three?

### ISSUE THREAD

**Issue by dialpad-drew** [dialpad/dialtone#188]
Dialtone's XS, SM, and MD button sizes are really similar (28px, 30px, and 32px respectively). Are all three sizes needed and used? Is there an opportunity for potential consolidation here?

**Comment by hynes-dialpad** [comment]
Currently the buttons align with input XS, SM, and MD sizes (or they should at least). We could look at making these sizes more noticeable, but we would have to make they're in line with inputs for situations where inputs and button are side-by-side.

**Comment by dialpad-drew** [comment]
Good call, Josh. I actually did these last night and am going to save them here for archival purposes.

Audit numbers as of March 9:
<img width="957" alt="Buttons" src="https://user-images.githubusercontent.com/77751297/110667118-2cf8c100-8187-11eb-8f99-1db7e648b7c1.png">
![Selects](https://user-images.githubusercontent.com/77751297/110667123-2f5b1b00-8187-11eb-905d-ddf3dc9018f9.png)
![Text Inputs](https://user-images.githubusercontent.com/77751297/110667127-2f5b1b00-8187-11eb-9bbc-2ba42ab840dd.png)
![Textareas](https://user-images.githubusercontent.com/77751297/110667128-2ff3b180-8187-11eb-895c-01354dfbef5c.png)

**Comment by dialpad-drew** [comment]
![Screen Shot 2021-03-10 at 6 10 10 PM](https://user-images.githubusercontent.com/77751297/110720709-4bcd7680-81cc-11eb-8150-6c8c15d1c1c3.png)
![Screen Shot 2021-03-10 at 6 10 46 PM](https://user-images.githubusercontent.com/77751297/110720713-4cfea380-81cc-11eb-8f3f-c3757bfa7376.png)
![Screen Shot 2021-03-10 at 6 11 05 PM](https://user-images.githubusercontent.com/77751297/110720716-4cfea380-81cc-11eb-8722-ac0063618ad2.png)
![Screen Shot 2021-03-10 at 6 11 28 PM](https://user-images.githubusercontent.com/77751297/110720720-4d973a00-81cc-11eb-8f74-4cb2a4320616.png)
